src/core/util/util.spec.js

```diff
@@ -223,10 +223,33 @@ describe('util', function() {
         // Restore the scrolling.
         enableScrolling();
         window.scrollTo(0, 0);
-        document.body.removeChild(element[0]);
+
+        element.remove();
       }));
     });
 
+    describe('getViewportTop', function() {
```

Comment:
Can we add some tests to make sure everything is correct even if they use `<md-content>` to scroll (rather than the body)?